### ISSUE TITLE
Issue with feedback lines

### ISSUE THREAD

**Issue by swilliamson7** [EnzymeAD/Enzyme.jl#1045]
@wsmoses This is the bug we saw when I include the feedback lines which effectively just share things such as model integration time, NaN checks, etc. 

[feedback_error.txt](https://github.com/EnzymeAD/Enzyme.jl/files/12579250/feedback_error.txt)

**Comment by wsmoses** [comment]
Yeah this one requires some minimization, if possible. Can you also post
the source code reproducer?

**Comment by wsmoses** [comment]
@swilliamson7 any chance you have a reproducer for this?

**Comment by wsmoses** [comment]
bumping this @swilliamson7

**Comment by swilliamson7** [comment]
Sorry for the delay! I minimized a bit, and can try working on it more later 

Running [this script](https://github.com/swilliamson7/ShallowWaters.jl/blob/zannabolton/my_scripts/output_error.jl) will trigger Enzyme bugs seemingly related to lines that just provide feedback on the integration or initialize netcdf output

**Comment by wsmoses** [comment]
@swilliamson7 does this still err?

**Comment by swilliamson7** [comment]
Last I checked yes

**Comment by wsmoses** [comment]
@swilliamson7 I don't see a file at the linked script. Do you have something that still errs [we also since fixed a bunch of things]

**Comment by swilliamson7** [comment]
Yeah things have been moved around since May. I think I still get this bug, but have just been omitting the lines of code that cause it. We can either (a) leave this open and in the coming days I'll create a new script that leads to it or (b) close this issue and I'll reopen whenever I get around to looking at it more

**Comment by wsmoses** [comment]
just wanted to circle back if this is still an issue

**Comment by wsmoses** [comment]
closing as resolved
